compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -1485,6 +1485,13 @@ object Types {
      */
     def signature(implicit ctx: Context): Signature = Signature.NotAMethod
 
+    def dropRepeatedAnnot(implicit ctx: Context): Type = this match {
+      case AnnotatedType(parent, annot) if annot.symbol eq defn.RepeatedAnnot => parent
+      case tp @ AnnotatedType(parent, annot) =>
+        tp.derivedAnnotatedType(parent.dropRepeatedAnnot.stripPoly, annot)
```

Comment:
Why the .stripPoly ?

Reply:
That is a very good question and my answer is: I don't know! I must have accidentally pasted this code from somewhere when committing.